holmes/plugins/toolsets/newrelic/newrelic.py

```
@@ -0,0 +1,425 @@
+            "start": rfc3339(int(start_ts)) if start_ts is not None else "",
+            "end": rfc3339(int(end_ts)) if end_ts is not None else "",
+            "step": int(step),
+            "output_type": params.get("output_type", "Plain"),
+            "data": {"resultType": "matrix", "result": prom_series},
+        }
+        return response_data
+
+    # 2) Update _invoke to use the Prometheus-style formatter for metrics / TIMESERIES
+    #    (keep your existing logs path as-is)
+    def _invoke(
+        self, params: dict, user_approved: bool = False
```

Ruff (0.12.2)

Unused method argument: `user_approved`

(ARG002)